src/czml3/properties.py

```diff
@@ -142,6 +142,20 @@ def is_valid(cls, color):
             and (3 <= len(color) <= 4)
         ):
             return all(0 <= v <= 1 for v in color)
+        # (R, G, B) or (R, G, B, A)
+        if (
+            isinstance(color, tuple)
```

Comment:
If this logic is exactly the same as lines 132-137, can we merge them? Something like:

```
python
if (
    isinstance(color, (list, tuple))
    and ...
):
    return ...
```

Reply:
makes sense. Will have a go at it.